sktime/tests/_config.py

```diff
@@ -281,16 +279,7 @@
 # We use estimator tags in addition to class hierarchies to further distinguish
 # estimators into different categories. This is useful for defining and running
 # common tests for estimators with the same tags.
-VALID_ESTIMATOR_TAGS = (
-    "fit-in-transform",  # fitted in transform or non-fittable
-    "univariate-only",
-    "transform-returns-same-time-index",
-    "handles-missing-data",
-    "skip-inverse-transform",
-    "requires-fh-in-fit",
-    "X-y-must-have-same-index",
-    "enforce-index-type",
-)
+VALID_ESTIMATOR_TAGS = tuple(ESTIMATOR_TAG_LIST)
```

Comment:
Since we just wrap them in tuples here now, should we perhaps rename `VALID_ESTIMATOR_TAGS` to `ESTIMATOR_TAG_LIST` everywhere (and the same for the other variables)? Could also do that in a separate PR

Reply:
I'm using the refactor principle "avoid proliferation of scope" which means to keep changes localized behind natural interfaces or localized at some semantic principle. That is in general safer, if you don't refactor the monkey with the banana and the entire jungle, for using a `sklearn` metaphor.

I'm only changing how the variables are imported/created, not how they are used. Your suggestion would additionally require to ensure that the refactor would have to go carefully through the entire testing module to ensure all calls to `VALID_ESTIMATOR_TAGS` don't break.

Variable renaming and/or type change is a textbook candidate for a single change PR.

Reply:
so, yes, why not, but in a separate PR?